index.js

```diff
@@ -758,23 +769,11 @@ let applyInput = (input) => {
   }
 
   const [command, ...args] = values;
-  const room = getRoom(disk.roomId);
-
+  
   if (args.length === 1) {
     exec(commands[1][command], args[0]);
-  } else if (command === 'take' && args.length) {
-    // support for taking items with spaces in the names
-    // (just tries to match on the first word)
-    takeItem(args[0]);
-  } else if (command === 'use' && args.length) {
-    // support for using items with spaces in the names
-    // (just tries to match on the first word)
-    useItem(args[0]);
   } else if (args.length >= commands.length) {
     exec(commands[commands.length - 1][command], args);
-  } else if (room.exits && getExit(command, room.exits)) {
-    // handle shorthand direction command, e.g. "EAST" instead of "GO EAST"
```

Comment:
I appreciate what you're trying to do here, but this code currently handles not just cardinal directions but custom exits. For instance, play the demo disk and type `north` followed by `up` to go upstairs.

Reply:
Ah yes, that makes sense! That's a nice shorthand. I'll revert this portion